script/dolphinscheduler-daemon.sh

```diff
@@ -81,8 +79,26 @@ else
   exit 1
 fi
 
+state=""
+function get_server_running_status() {
+  state="STOP"
+  if [ -f $pid ]; then
+    TARGET_PID=`cat $pid`
+    server_running_count=`ps -p $TARGET_PID --no-headers | wc -l`
```

Comment:
It seems like `--no-headers` can not be running in `MacOS`. It's better to use a compatible way.

Reply:
> It seems like `--no-headers` can not be running in `MacOS`. It's better to use a compatible way.

@SbloodyS Thanks a lot for pointing out that, I have removed `--ho-headers`.